azure_functions_worker/constants.py

```diff
@@ -55,3 +55,6 @@
 
 # Paths
 CUSTOMER_PACKAGES_PATH = "/home/site/wwwroot/.python_packages/lib/site-packages"
+
+# Flag to index functions in handle init request
+ENABLE_INIT_INDEXING = "ENABLE_INIT_INDEXING"
```

Comment:
confirming my understanding, by default is this set to false? if we verify this is working once released, we will remove this to keep customer transparent as this is more related to worker's internal mechanism?

Reply:
if this is a public app setting, prefix with PYTHON_?

Reply:
also this is checked in handle envreload request too, maybe PYTHON_ENABLE_ON_START_INDEXING or PYTHON_ENABLE_FAST_INDEXING?

Reply:
Right, this will be set to false by default. Will started enabling by stamp and will be removed later. This is not exposed to the public. Also FAST_INDEXING and START_INDEXING doesn't seem accurate to this use case. ENABLE_INIT_INDEXING is more apt IMO